packages/plugin-core/src/commands/CreateNoteCommand.ts

```diff
@@ -8,10 +8,10 @@ import {
 import { AutoCompletableRegistrar } from "../utils/registers/AutoCompletableRegistrar";
 import _ from "lodash";
 import { window } from "vscode";
-import { NoteProps, NoteUtils } from "@dendronhq/common-all";
 import { AnalyticsUtils } from "../utils/analytics";
+import { ExtensionProvider } from "../ExtensionProvider";
 
-type CommandOpts = NoteProps & {};
+type CommandOpts = string & {};
```

Comment:
The suffix `Opts` means to me option-object than a scalar value. Maybe we should keep it being an option-object?